ci/pipeline.yml

```diff
@@ -406,6 +402,22 @@ jobs:
       - terraform-yaml/state.yml
       - common-tooling/tooling-bosh.yml
       ops_files:
+      - bosh-deployment/uaa.yml
```

Comment:
What do these `ops_files` add?

Reply:
It's a [yaml merge syntax](https://bosh.io/docs/cli-ops-files/), and this is the way that functionality is exposed in the bosh deployment resource in Concourse.